packages/api/src/hooks/p2p/entity/counterparty/p2p-advertiser-relations/index.ts

```diff
@@ -0,0 +1,3 @@
+export { default as useGet } from './useAdvertiserRelations';
+export { default as useBlock } from './useAdvertiserRelationsAddBlocked';
+export { default as useUnblock } from './useAdvertiserRelationsRemoveBlocked';
```

Comment:
just a question, if we name the function like this `useBlockAdvertiser` is it too much since its already under the entity of p2p-advertiser-relations?

Reply:
wouldn't it be kind of a repetition?

Reply:
yeah true true